common/config/rush/browser-approved-packages.json

```diff
@@ -130,6 +130,10 @@
       "name": "@gooddata/sdk-model",
       "allowedCategories": [ "examples", "production", "tools" ]
     },
+    {
+      "name": "@gooddata/sdk-theme-provider",

```

Comment:
This is probably obsolete, there is no `@gooddata/sdk-theme-provider` anymore. Please remove this entry.